exercise/curry.js

```diff
@@ -0,0 +1,59 @@
+/*
+## Curry
+
+### Instruction
+
+Create a function `curry` that receives as parameters a string and a
```

Comment:
it's wierd that the `curry` function does not return a function.

I don't think we want to call this one curry.